src/types/index.ts

```diff
@@ -27,6 +27,14 @@ export type PatchScheduleRequest =
   temporal.api.workflowservice.v1.IPatchScheduleRequest;
 export type UpdateScheduleRequest =
   temporal.api.workflowservice.v1.IUpdateScheduleRequest;
+export type FailActivityTaskRequest =
```

Comment:
This file is auto-generated, so we might want to put this in a `.d.ts` file elsewhere.